### DISCUSSION TITLE
Scrolling text on only partial display

### DISCUSSION THREAD

**Discussion by agganitk** [board707/DMD_STM32#102] [General]
Hi
I am scrolling the text towards left in  P10 display. Board has multiple P10 connected in series.  I do not want the text to reach towards left edge of board. I want to use only partial display for scrolling the text. Can you suggest please?

**Comment by board707** [comment]
Hi
It can't be achieved with Marquee methods.
But you can do it "manually" by redrawing the text in a new position every 30mS, for example.

**Comment by agganitk** [comment]
thanks..
